src/common/cm/cm_test.cpp

```diff
@@ -247,19 +244,22 @@ int CM_PointContents( const vec3_t p, clipHandle_t model )
 		// XreaL END
 
 		// see if the point is in the brush
-		for ( i = 0; i < b->numsides; i++ )
+		const cbrushside_t *firstSide = b->sides;
+		const cbrushside_t *endSide = firstSide + b->numsides;
+		const cbrushside_t *side;
+		for ( side = firstSide; side < endSide; side++ )
 		{
-			d = DotProduct( p, b->sides[ i ].plane->normal );
+			float d = DotProduct( p, side->plane->normal );
 
 // FIXME test for Cash
-//          if ( d >= b->sides[i].plane->dist ) {
-			if ( d > b->sides[ i ].plane->dist )
+//          if ( d >= side->plane->dist ) {
```

Comment:
maybe use the occasion to kill zombi code, too.

Reply:
From the comment I actually don't know if the zombie code is wrong, or if, on the contrary, the zombie code is the expected correct one and the non-commented one a workaround for a crash, for example.
Someone wrote a `FIXME` for a reason, I won't touch it.